pkgs/tools/X11/arandr/default.nix

```diff
@@ -1,22 +1,26 @@
-{ lib, fetchurl, python3Packages
-, gobject-introspection, gsettings-desktop-schemas, gtk3
-, wrapGAppsHook, xrandr
+{ lib
+, fetchurl
+, python3Packages
+, gobject-introspection
+, gsettings-desktop-schemas
+, gtk3
+, wrapGAppsHook
+, xrandr
 }:
 
 let
   inherit (python3Packages) buildPythonApplication docutils pygobject3;
+in
+buildPythonApplication rec {
   pname = "arandr";
   version = "0.1.10";
-  name = "${pname}-${version}";
-in buildPythonApplication rec {
-  inherit pname version;
```

Comment:
if these had been kept in the let block + the inherit, it would have been possible to get rid of the rec in the buildPythonApplication attrset

Reply:
What is the advantage in this case though? `rec` is doing exactly the same thing, and I think the only real reason that `rec` is polemic is because during `overrideAttrs` it doesn't work as many people expect it. However keeping the `inherit` wouldn't work either.

Reply:
the advantage would just be dropping rec. If keeping it works better in cases of override then it's not worth removing it

Reply:
> If keeping it works better in cases of override then it's not worth removing it

My point is: it doesn't.

The only way to make it easier to override is using the new `mkDerivation (finalAttrs: {})` introduced a few weeks ago, but this is still kinda of not recommended since we still are not sure if this is the way to go (from what I remember).

Reply:
ack, I misread the original comment, sorry. Then there is nothing much to be done here and this change is an improvement either way